CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ## Other
 
 - Work around build failures when building `bat` from vendored sources #3179 (@dtolnay)
+- CICD: Stop building for x86_64-pc-windows-gnu which fails #3261 (Enselic)
```

Comment:
(Not having @ in front of my name is intentional, because I want to avoid mentions by tools.)